piksi_tools/console/settings_view.py

```diff
@@ -170,10 +170,33 @@ def revert_to_prior_value(self, name, old, new):
         invalid_setting_prompt = prompt.CallbackPrompt(
             title="Settings Write Error",
             actions=[prompt.close_button], )
-        invalid_setting_prompt.text = \
-            ("\n   Unable to confirm that {0} was set to {1}.\n"
-             "   Ensure the range and formatting of the entry are correct.\n"
-             "   Ensure that the new setting value did not interrupt console communication.").format(self.name, new)
+        if error_value == 1:    # SBP_WRITE_STATUS_VALUE_REJECTED
```

Comment:
Let's define these values near the top of the module somewhere (i.e. try to reduce magic numbers)